docs/_docs/thin-clients/java-thin-client.adoc

```diff
@@ -132,7 +132,7 @@ include::{sourceCodeFile}[tag=key-value-operations,indent=0]
 -------------------------------------------------------------------------------
 
 === Executing Scan Queries
-Use the `ScanQuery<K, V>` class to get a set of entries that satisfy a given condition. The thin client sends the query to the cluster node where it is executed as a normal link:key-value-api/using-scan-queries[scan query].
+Use the `ScanQuery<K, V>` class to get a set of entries that satisfy a given condition. The thin client sends the query to the cluster node where it is executed as a normal link:key-value-api/using-cache-queries[scan query].
```

Comment:
How about:"...where it is executed as a **regular** link:key-value-api/using-cache-queries[scan query]."

Reply:
@Nikita-tech-writer hi, thanks a lot for review. I submitted you suggestion.

Reply:
@timoninmaxim Perfect! Then the PR can be merged and ported to 2.11 branch (to publish the changes on the website).

Reply:
Hi! This docs are for the feature that is available since 2.12 only. So, no need to cherry-pick this back.